seccomp.go

```diff
@@ -756,6 +777,18 @@ func (f *ScmpFilter) SetNoNewPrivsBit(state bool) error {
 	return f.setFilterAttr(filterAttrNNP, toSet)
 }
 
+// SetLogBit sets the state of the Log bit, which will be applied on filter
+// load, or an error if an issue was encountered setting the value.
```

Comment:
Same as above, please add something about minimum kernel version where this will take effect. And/or minimum libseccomp version.

Reply:
Sounds good.

Reply:
In the update to this PR, I've added a comment about the minimum API level required (3).